langchain/src/vectorstores/chroma.ts

```diff
@@ -114,8 +114,10 @@ export class Chroma extends VectorStore {
     texts: string[],
     metadatas: object[],
     embeddings: Embeddings,
-    collectionName?: string,
-    url?: string
+    dbConfig: {
+      collectionName?: string;
+      url?: string;
+    }
```

Comment:
i had to change the type signature of the `fromTexts` methods of these vector stores in order to standardize them enough for the base class, in order to use `fromTexts` in example selectors. idk if other docs need to be updated, or if they need to be notified

Reply:
I can have a look at the docs

Reply:
@sw-yx I'll be reverting this change as since you opened the PR a different fix for this issue has been committed to main

Reply:
Actually will keep this, thank you. Will just update the docs in a separate PR